public/index.html

```diff
@@ -3,24 +3,81 @@
   <head>
     <meta charset="utf-8">
     <title>Hoodie</title>
-  </head>
-  <body>
-    <h1>Hello from Hoodie!</h1>
+    <style>
```

Comment:
Yeah but it still makes sense to have them with a uniform style maybe a **global** css where both `/hoodie/*`and the message that shows you there is no `/public`folder have a uniform branding and style.